.github/workflows/after-pullrequest.yml

```diff
@@ -20,9 +20,8 @@ jobs:
     if: github.event.workflow_run.conclusion != 'skipped'
 
     permissions:
-      actions: write
+      checks: write
       pull-requests: write
-      # actions: read
       # only needed for private repository
```

Comment:
That's strange, this change is already in `main`. Why is GitHub showing it as changed again? What we have above is correct, so there is nothing to worry about.